lib/move.js

```diff
@@ -27,8 +29,25 @@ const moveSync = (from, to) => {
     fs.renameSync(from, to);
   } catch (err) {
     if (err.code !== "ENOENT") {
```

Comment:
Instead of:

```js
if (err.code !== "ENOENT") {
	if (err.code === "EXDEV") {
		...1
	} else {
		...2
	}
} else {
	...3
}
```

we can refactor the code so that the if statements always use `===`:

```js
if (err.code === "ENOENT") {
	...3
} else if (err.code === "EXDEV") {
	...1
} else {
	...2
}
```

Reply:
Thank you!! Done!